### PR TITLE
[SMALLFIX] Reset heartbeat pools

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

**Comment by peisun1115** [comment]
@apc999 @ifcharming

**Comment by peisun1115** [comment]
@jsimsa

File: core/client/src/main/java/alluxio/client/file/FileSystemContext.java [hunk 1]
```diff
@@ -164,10 +164,22 @@ public void close() {
       pool.close();
     }
     mBlockWorkerClientPools.clear();
+
+    for (BlockWorkerThriftClientPool pool : mBlockWorkerHeartbeatClientPools.values()) {
```
Comment (jsimsa) [inline core/client/src/main/java/alluxio/client/file/FileSystemContext.java]: can we use consistent names

`mBlockWorkerHeartbeatClientPools` vs `mFileSystemWorkerClientHeartbeatPools`

Comment (peisun1115) [inline core/client/src/main/java/alluxio/client/file/FileSystemContext.java]: Done.

**Comment by peisun1115** [comment]
@jsimsa PTAL

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**